### ISSUE TITLE
Facing Problem while installing Probe

### ISSUE THREAD

**Issue by MayankSainiTk20** [psi-probe/psi-probe#749]
I am getting the below error while installing probe with my application

probe version: 2.4
tomcat version: 7.X

The same is working fine when i install the app on a fresh tomcat

SEVERE: Exception sending context initialized event to listener instance of class org.springframework.web.context.ContextLoaderListener
java.lang.IllegalStateException: Level number 20 is not recognized.
        at org.slf4j.impl.Log4jLoggerAdapter.log(Log4jLoggerAdapter.java:421)
        at org.apache.commons.logging.impl.SLF4JLocationAwareLog.info(SLF4JLocationAwareLog.java:120)
        at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:189)
        at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:45)
        at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4723)
        at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5226)
        at org.apache.catalina.core.StandardContext$1.call(StandardContext.java:5221)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)

**Comment by hazendaz** [comment]
That appears to be a slf4j issue.  Are you using slf4j elsewhere?

**Comment by MayankSainiTk20** [comment]
Yes slf4j(slf4j-log4j12-1.3.0.jar slf4j-api-1.3.0.jar) is used in my application and it is getting loaded from common lib directory of tomcat  while in probe  1.4.2 version of jars are used

**Comment by MayankSainiTk20** [comment]
I have removed slf4j-log4j12-1.3.0.jar and slf4j-api-1.3.0.jar jars from my application and now only 1.4.2 version of jars are getting loaded but still there is no success. I started getting new error java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory

**Comment by hazendaz** [comment]
Try using the same version in your common loader.  Ive seen that before and it was failing due to conflicting versions.  Using tomcat the way you are needs the jars in common loader.

Get Outlook for Android<https://aka.ms/ghei36>



On Tue, Jul 19, 2016 at 3:26 AM -0400, "Mayank Saini" <notifications@github.com<mailto:notifications@github.com>> wrote:


I have removed slf4j-log4j12-1.3.0.jar and slf4j-api-1.3.0.jar jars from my application and now only 1.4.2 version of jars are getting loaded but still there is no success. I started getting new error java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory

-
